test/Microsoft.AspNet.Mvc.Extensions.Test/ViewComponents/JsonViewComponentResultTest.cs

```diff
@@ -18,29 +18,12 @@ namespace Microsoft.AspNet.Mvc
     public class JsonViewComponentResultTest
     {
         [Fact]
-        public void Execute_SerializesData_UsingSpecifiedFormatter()
-        {
-            // Arrange
-            var view = Mock.Of<IView>();
-            var buffer = new MemoryStream();
-            var viewComponentContext = GetViewComponentContext(view, buffer);
-
-            var expectedFormatter = new JsonOutputFormatter();
-            var result = new JsonViewComponentResult(1, expectedFormatter);
-
-            // Act
-            result.Execute(viewComponentContext);
-            buffer.Position = 0;
-
-            // Assert
-            Assert.Equal(expectedFormatter, result.Formatter);
-            Assert.Equal("1", new StreamReader(buffer).ReadToEnd());
-        }
-
-        [Fact]
         public void Execute_UsesFormatter_WithSpecifiedSerializerSettings()
         {
             // Arrange
+            var abcdIndentedUTF8Bytes
```

Comment:
Could you use strings instead? `Assert.Equal(abcdIndented, Encoding.UTF8.GetString(body.ToArray())` - easier to understand what's happening